cypress/e2e/cloud/helpBar.test.ts

```diff
@@ -60,10 +64,16 @@ describe('Help bar support for PAYG users', () => {
     cy.getByTestID('payg-support-overlay-header').should('exist')
 
     cy.getByTestID('contact-support-subject-input').clear().type(subject)
-    cy.getByTestID('dropdown--button').click()
-    cy.getByTitle('1 - Critical').click()
-
-    cy.getByTestID('support-description--textarea').clear().type(description)
-    cy.getByTestID('payg-contact-support--submit').should('not.be.disabled')
+    cy.getByTestID('severity-level-dropdown')
+      .within(() => {
```

Comment:
This is necessary because, with multi-org on, there's more than one dropdown on the page, so clicking on something with a testID of 'dropdown--button' isn't sufficiently specific anymore.